flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/expressions/ExpressionSerializationTest.java

```diff
@@ -172,13 +172,13 @@ public static Stream<TestSpec> testData() {
                         .expectStr("POSITION(`f0` IN 'ABC')"),
                 TestSpec.forExpr($("f0").trim("ABC"))
                         .withField("f0", DataTypes.STRING())
-                        .expectStr("TRIM BOTH 'ABC' FROM `f0`"),
+                        .expectStr("TRIM(BOTH 'ABC' FROM `f0`)"),
                 TestSpec.forExpr($("f0").trimLeading("ABC"))
                         .withField("f0", DataTypes.STRING())
-                        .expectStr("TRIM LEADING 'ABC' FROM `f0`"),
+                        .expectStr("TRIM(LEADING 'ABC' FROM `f0`)"),
                 TestSpec.forExpr($("f0").trimTrailing("ABC"))
                         .withField("f0", DataTypes.STRING())
-                        .expectStr("TRIM TRAILING 'ABC' FROM `f0`"),
+                        .expectStr("TRIM(TRAILING 'ABC' FROM `f0`)"),
                 TestSpec.forExpr($("f0").overlay("ABC", 2))
```

Comment:
I am looking at the docs [https://nightlies.apache.org/flink/flink-docs-release-1.20/docs/dev/table/functions/systemfunctions/](https://nightlies.apache.org/flink/flink-docs-release-1.20/docs/dev/table/functions/systemfunctions/) I see the syntax as `STRING1.trim(LEADING, STRING2)` and the like - I am not seeing FROM syntax I see here. Or is this syntax part of the table planner representation of this function.

Reply:
there are sql and table api
which of them are you talking about?
sql is in left column in contains
```sql
TRIM([ BOTH | LEADING | TRAILING ] string1 FROM string2) 
```
table api among others contains what you've mentioned